{{cookiecutter.project_name}}/src/{{cookiecutter.package_name}}/pp/basic.py

```diff
@@ -15,3 +24,68 @@ def basic_preproc(adata: AnnData) -> int:
     """
     print("Implement a preprocessing function here.")
     return 0
+
+
+def elaborate_example(
+    items: Iterable[ScverseDataStructures],
+    transform: Callable[[Any], str],
+    /,  # indicates the end of positional only arguments
+    *,  # functions after the asterix are key word only arguments
+    layer_key: str | None = None,
+    mudata_mod: str | None = "rna",  # Only specify defaults in the signature, not the docstring!
+    sdata_table_key: str | None = "table1",
+    max_items: int = 100,
+) -> list[Any]:
```

Comment:
given the type of `transform` this would return a list of `str` wouldn't it?

Reply:
Ah right, I had it earlier and then forgot that I had specified the transform type more narrowly. Thanks